src/guards/book-exists.ts

```diff
@@ -50,7 +49,7 @@ export class BookExistsGuard implements CanActivate {
    */
   hasBookInApi(id: string) {
     return this.googleBooks.retrieveBook(id)
-      .map(book => this.bookActions.loadBook(book))
+      .map(book => new bookActions.LoadBook(book))
       .do(action => this.store.dispatch(action))
       .map(book => !!book)
       .catch(() => {
```

Comment:
Then the comment should be fixed to make that clear.  Thanks for the clarification!